docs/docs/micromatch-wrapper.md

```diff
@@ -0,0 +1,52 @@
+#### Overview
+[Micromatch](https://github.com/micromatch/micromatch) support various matching features, such as glob patterns and advanced globbing
+
+Micromatch is a Node.js application, while MBT is a Go application. The micromatch wrapper is a package that enables MBT to use the functionalities provided by micromatch. From MBT 1.2.25 version, we create [micromath wrapper](https://github.com/SAP/cloud-mta-build-tool/tree/master/micromatch) to package it.

```

Comment:
From the MBT 1.2.25 version, we provide the [micromatch wrapper](https://github.com/SAP/cloud-mta-build-tool/tree/master/micromatch) to package it.

Reply:
OK, Done